### PR TITLE
XSRF token error when switching environments through the same TCP port

### PR DESCRIPTION
**Pull request by Dannyps**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

# Description
This is a bug that affects only the deployment process as far as we can see. The token from environment A gets used on calls to env B after the active env change, which leads to an (expected) error. What we did was detect such an error and reload the relevant tokens.

### CODE REVIEW

File: AdminUi/src/AdminUi/Controllers/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class RedirectAntiforgeryValidationFailedResultFilter : IAlwaysRunResultFilter
+{
+    public void OnResultExecuting(ResultExecutingContext context)
+    {
+        if (context.Result is ObjectResult result
+            && result.StatusCode == 400
+            && result.Value is ProblemDetails problemDetails
+            )
+        {
+            problemDetails.Detail = "xsrf-token-may-be-invalid";
```
Comment (tnotheis) [inline AdminUi/src/AdminUi/Controllers/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs:15]: Aren't we hiding some potentially important information by overriding the details?

**Review by tnotheis — CHANGES_REQUESTED** [review]

**Comment by tnotheis** [comment]
@Dannyps What about this PR?

File: AdminUi/src/AdminUi/Controllers/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs (hunk 1 above)
Comment (Dannyps) [reply AdminUi/src/AdminUi/Controllers/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs:15]: In all my attempts, `problemDetails.Detail` has been null. I can add a check for that, however, if you consider that preferable.

Comment (tnotheis) [reply AdminUi/src/AdminUi/Controllers/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs:15]: Okay, I'm fine with leaving it as it is then.

**Review by tnotheis — APPROVED** [review]

**Pull request merged**